src/test/transaction_tests.cpp

```diff
@@ -531,9 +529,29 @@ BOOST_AUTO_TEST_CASE(test_simple_joinsplit_invalidity)
     }
 }
 
-BOOST_AUTO_TEST_CASE(test_Get)
+BOOST_AUTO_TEST_CASE(test_simple_joinsplit_invalidity_driver) {
+    {
+        CMutableTransaction mtx;
+        mtx.nVersion = 2;
+        test_simple_joinsplit_invalidity(SPROUT_BRANCH_ID, mtx);
+    }
+    {
+        CMutableTransaction mtx;
+        mtx.fOverwintered = true;
+        mtx.nVersionGroupId = OVERWINTER_VERSION_GROUP_ID;
+        mtx.nVersion = 3;
+        const Consensus::Params& params = Params().GetConsensus();
+        int tmp = params.vUpgrades[Consensus::UPGRADE_OVERWINTER].nActivationHeight;
+        const_cast< Consensus::Params&>(params).vUpgrades[Consensus::UPGRADE_OVERWINTER].nActivationHeight = Consensus::NetworkUpgrade::ALWAYS_ACTIVE;
+        test_simple_joinsplit_invalidity(NetworkUpgradeInfo[Consensus::UPGRADE_OVERWINTER].nBranchId, mtx);
+        const_cast< Consensus::Params&>(params).vUpgrades[Consensus::UPGRADE_OVERWINTER].nActivationHeight = tmp;
```

Comment:
Use regtest params, and replace the above with:

```cpp
UpdateNetworkUpgradeParameters(Consensus::UPGRADE_OVERWINTER, Consensus::NetworkUpgrade::ALWAYS_ACTIVE);
test_simple_joinsplit_invalidity(NetworkUpgradeInfo[Consensus::UPGRADE_OVERWINTER].nBranchId, mtx);
UpdateNetworkUpgradeParameters(Consensus::UPGRADE_OVERWINTER, Consensus::NetworkUpgrade::NO_ACTIVATION_HEIGHT);
```

Reply:
Updated and pushed.